### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by younik**
Changes after running `pre-commit run --all-files`, which fixes `black`, `flake8`, `autoflake` and `isort`.

(Another step towards https://github.com/GFNOrg/torchgfn/pull/193 to make easier focusing on pyright issues)

### CODE REVIEW

File: tutorials/examples/train_ising.py [hunk 1]
```diff
@@ -81,10 +78,10 @@ def ising_n_to_ij(L, n):
     optimizer = torch.optim.Adam(gflownet.parameters(), lr=1e-3)
 
     # Learning
-    visited_terminating_states = env.States.from_batch_shape((0,))
+    env.States.from_batch_shape((0,))
```
Comment (josephdviviano) [inline tutorials/examples/train_ising.py:81]: Does this do anything? It should be assigned.

Did you test all of the scripts (see `examples/test_scripts.py`)?

File: tutorials/examples/train_ising.py [hunk 2]
```diff
@@ -26,9 +26,6 @@ def main(args):
 
     n_hidden = 2
     acc_fn = "relu"
-    lr = 0.001
-    lr_Z = 0.01
-    validation_samples = 1000
```
Comment (josephdviviano) [inline tutorials/examples/train_ising.py:31]: Don't do this please. Variables should be named somehow to make it easier for the reader. I don't like magic numbers hanging around in scripts -- it requires more work to understand what they are for.

**Review by josephdviviano — CHANGES_REQUESTED** [review]
Great - just one or two small questions :)

File: tutorials/examples/train_ising.py (hunk 2 above)
Comment (younik) [reply tutorials/examples/train_ising.py:31]: These variables are not used

Only `lr` could be inserted at line 73, but it has a different value, and I believe it is clearer as it is:
```python
optimizer = torch.optim.Adam(gflownet.parameters(), lr=1e-3)
```

File: tutorials/examples/train_ising.py (hunk 1 above)
Comment (younik) [reply tutorials/examples/train_ising.py:81]: `visited_terminating_states` is not used. Actually, the line itself is useless, so I should remove it instead(?)
Tests are working fine in both cases

Comment (younik) [reply tutorials/examples/train_ising.py]: Actually this line was removed in https://github.com/GFNOrg/torchgfn/pull/203

File: tutorials/examples/train_ising.py (hunk 2 above)
Comment (younik) [reply tutorials/examples/train_ising.py]: Actually, https://github.com/GFNOrg/torchgfn/pull/203 inserted lr in that line
(what are the odds)

Comment (josephdviviano) [reply tutorials/examples/train_ising.py]: OK - I must have lost track - thanks for following up on this.

File: tutorials/examples/train_ising.py (hunk 1 above)
Comment (josephdviviano) [reply tutorials/examples/train_ising.py]: OK makes sense to remove it then! Thanks

**Review by josephdviviano — APPROVED** [review]
lgtm!

**Pull request merged**